init-tools.sh

```diff
@@ -9,6 +9,7 @@ __DOTNET_CMD=$__DOTNET_PATH/dotnet
 if [ -z "$__BUILDTOOLS_SOURCE" ]; then __BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json; fi
 export __BUILDTOOLS_USE_CSPROJ=true
 __BUILD_TOOLS_PACKAGE_VERSION=$(cat $__scriptpath/BuildToolsVersion.txt)
+if [ -z "$__ILLINK_SOURCE" ]; then __ILLINK_SOURCE=https://dotnet.myget.org/F/dotnet-core/api/v3/index.json; fi
```

Comment:
Same for this script we should avoid adding more feeds to restore the tools. All the dependencies should come from dotnet-buildtools feed.